lib/performance_data/statistics.rb

```diff
@@ -191,7 +189,7 @@ def get_part_urls(content, slug)
 
       parts = details.fetch("parts")
 
-      part_urls = parts.map!{ |part| "#{slug}/#{part['slug']}" }
+      part_urls = parts.map! { |part| "#{slug}/#{part['slug']}" }
```

Comment:
Minor nitpick, but could we rebase this whitespace change into the previous commit?